simtools/corsika/corsika_histograms.py

```diff
@@ -1329,41 +1419,50 @@ def fill_ecsv_table(self, hist, x_edges, y_edges, x_label, y_label):
             The x edges of the histograms.
         y_edges: numpy.array
             The y edges of the histograms.
+            Use None if dealing with 1D histogram.
         x_label: str
             X edges label.
         y_label: str
             Y edges label.
+            Use None if dealing with 1D histogram.
```

Comment:
Suggest to shorten this "Use None for 1D histogram."
(unclear otherwise what dealing is)

Reply:
Done